tools/ci/travis/travis-ci.sh

```diff
@@ -118,6 +118,9 @@ EOF
       AS='as' ASPP='gcc -c' \
       $configure_flags
     ;;
+  s390x)
```

Comment:
(this doesn't need to be a separate case - just make the `x64)` above `x64|s390x)`